cfpb/cfgov-refresh
# Ask cfpb autocomplete

Add autocomplete for Ask CFPB search

## Additions

- Autocomplete molecule js/less
- Autocomplete for Ask CFPB search bar


## Testing

1. Check out branch
2. Run gulp
3. Set up Ask if it isn't already running locally
    ```
    export DEPLOY_ENVIRONMENT=build
    ./drop-db.sh
    ./create-mysql-db.sh
    mysql -uroot v1 < ~/Downloads/production_django.sql
    python cfgov/manage.py migrate
    python cfgov/manage.py runscript migrate_knowledgebase
    ```
4. Navigate to [Ask page](http://localhost:8000/ask-cfpb/) and check that autocomplete works in the Ask search bar

## Screenshots

<img width="757" alt="screen shot 2017-06-06 at 7 04 20 pm" src="https://user-images.githubusercontent.com/778171/26859213-144ee750-4aeb-11e7-882a-d4208367c3f8.png">

## Notes

- I moved the ajax code to a utility that could also be used for the `FormSubmit` module

## Checklist
